components/CaseStatus/EditCaseStatusForm/ChooseEditCaseStatusForm.tsx

```diff
@@ -15,12 +15,21 @@ const ChooseEditCaseStatusForm: React.FC<{
 }> = ({ personId, caseStatusId, prefilledValue, caseStatusType }) => {
   const router = useRouter();
 
-  const form_fields = CASE_STATUS.steps[0].fields;
+  let form_fields: any;
+
+  caseStatusType == 'LAC'
+    ? (form_fields = CASE_STATUS.steps[1].fields)
+    : (form_fields = CASE_STATUS.steps[0].fields);
 
   form_fields.map((field: any) => {
     if (prefilledValue) {
       field.default = String(prefilledValue);
     }
+    if (field.choices && caseStatusType != 'LAC') {
+      field.choices = field.choices.filter(function (el: any) {
+        return el.value != 'update';
```

Comment:
This if block is not needed (as we're loading a different form for updating LAC, we don't need to filter out anything from the choices)

Reply:
Removed